cmd/snap/color.go

```diff
@@ -161,41 +169,57 @@ func fillerPublisher(esc *escapes) string {
 // * if the publisher's username and display name match, it's just the display
 //   name; otherwise, it'll include the username in parentheses
 //
-// * if the publisher is verified, it'll include a green check mark; otherwise,
+// * if the publisher is "starred" it'll include a yellow star; if the
+//   publisher is "verified", it'll include a green check mark; otherwise,
 //   it'll include a no-op escape sequence of the same length as the escape
-//   sequence used to make it green (this so that tabwriter gets things right).
+//   sequence used to make it colorful (this so that tabwriter gets things
+//   right).
 func longPublisher(esc *escapes, storeAccount *snap.StoreAccount) string {
+	color := esc.green
```

Comment:
doing this here, but then setting color = esc.green anyway below is a bit strange, I would have left the code for the nil case alone, and just defined color parallel to badge, no?